cmd/agent/app/integrations.go

```diff
@@ -427,65 +340,108 @@ func install(cmd *cobra.Command, args []string) error {
+		return fmt.Errorf("error installing wheel %s: %v", wheelPath, err)
 	}
 
-	// Run pip check to determine if the installed integration is compatible with the base check version
-	pipCheckErr := pipCheck(cachePath)
-	if pipCheckErr == nil {
-		err := moveConfigurationFilesOf(integration)
-		if err == nil {
-			fmt.Println(color.GreenString(fmt.Sprintf(
-				"Successfully installed %s %s", integration, versionToInstall,
-			)))
-			return nil
-		}
-
+	// Move configuration files
+	if err := moveConfigurationFilesOf(integration); err != nil {
 		fmt.Printf("Installed %s %s", integration, versionToInstall)
 		return fmt.Errorf("Some errors prevented moving %s configuration files: %v", integration, err)
 	}
 
-	// We either detected a mismatch, or we failed to run pip check
-	// Either way, roll back the install and return the error
-	if currentVersion == nil {
-		// Special case where we tried to install a new integration, not yet released with the agent
-		pipArgs = []string{
-			"uninstall",
-			integration,
-			"-y",
-		}
-	} else {
-		pipArgs = append(pipArgs, fmt.Sprintf("%s==%s", integration, currentVersion))
+	fmt.Println(color.GreenString(fmt.Sprintf(
+		"Successfully installed %s %s", integration, versionToInstall,
+	)))
+	return nil
+}
+
+func downloadWheel(integration, version string) (string, error) {
+	pyPath, err := getCommandPython()
+	if err != nil {
+		return "", err
+	}
+	args := []string{
+		"-m", "datadog_checks.downloader",
+		integration,
+		"--version", version,
+	}
+	if verbose > 0 {
+		args = append(args, "-"+strings.Repeat("v", verbose))
 	}
+	downloaderCmd := exec.Command(pyPath, args...)
+	downloaderCmd.Env = os.Environ()
 
-	// Perform the rollback
-	pipErr := pip(pipArgs, false)
-	if pipErr == nil {
-		// Rollback successful, return error encountered during `pip check`
-		return fmt.Errorf(
-			"error when validating the agent's python environment, %s wasn't installed: %v",
-			integration, pipCheckErr,
+	// Proxy support
+	if err := common.SetupConfig(confFilePath); err != nil {
+		fmt.Printf("Cannot setup config, exiting: %v\n", err)
+		return "", err
+	}
+	proxies := config.GetProxies()
+	if proxies != nil {
+		downloaderCmd.Env = append(downloaderCmd.Env,
+			fmt.Sprintf("HTTP_PROXY=%s", proxies.HTTP),
+			fmt.Sprintf("HTTPS_PROXY=%s", proxies.HTTPS),
+			fmt.Sprintf("NO_PROXY=%s", strings.Join(proxies.NoProxy, ",")),
 		)
 	}
 
-	// Rollback failed, mention that the integration could be broken
-	return fmt.Errorf(
-		"error when validating the agent's python environment, and the rollback failed, so %s %s was installed and might be broken:\n - %v\n- %v",
-		integration, versionToInstall, pipCheckErr, pipErr,
-	)
+	// forward the standard error to the Agent logger
+	stderr, err := downloaderCmd.StderrPipe()
+	if err != nil {
+		return "", err
+	}
+	go func() {
+		in := bufio.NewScanner(stderr)
+		for in.Scan() {
+			fmt.Println(color.RedString(in.Text()))
+		}
+	}()
+
+	output, err := downloaderCmd.Output()
```

Comment:
This is a bit of an all or nothing approach with the output, would we get better feedback from the subprocess by doing what we're doing with `stderr` and read from the `downloaderCmd.StdoutPipe()` while executing the command with `downloaderCmd.Start()` `downloaderCmd.Wait()`? 

Of course this all probably only makes sense if we want to get a greater sense of progress from whatever the subprocess is spitting out, stuff like percentage downloaded, validation message, etc. etc (if they exist).